opensoundscape/utils.py

```diff
@@ -204,6 +206,15 @@ def generate_clip_times_df(
     return pd.DataFrame({"start_time": starts, "end_time": ends}).drop_duplicates()
 
 
+def cast_np_to_native(x):
```

Comment:
No tests for this function. I'm not sure tests would be necessary for a function this simple? But wanted to note.

Reply:
Maybe just to be on the safe side, the function should raise an error if passed something that is not a np datatype?

Reply:
added test